### ISSUE TITLE
Debug issues with mixpanel connector

### ISSUE THREAD

**Issue by aaronsteers** [airbytehq/PyAirbyte-Hackathon#9]
## Summary
The mixpanel source is having issues when run in PyAirbyte. The goal is to replicate and document any observed issues and then propose a fix.

## Resources to Assist
- PyAirbyte tutorials: https://github.com/airbytehq/quickstarts/tree/main/pyairbyte_notebooks
- PyAirbyte Documentaton: https://docs.airbyte.com/using-airbyte/pyairbyte/getting-started 
- PyAirbyte API Reference: https://docs.airbyte.com/using-airbyte/pyairbyte/getting-started#api-reference
- Source-linkedin-ads documentation: https://docs.airbyte.com/integrations/sources/linkedin-ads

**Comment by TLazarevic** [comment]
@aaronsteers 
I am interested in this one, with a team of additional two.

Also can you provide some sort of acceptance criteria (eg the expected output, should we write a readme page and a bug fix PR for example?)

**Comment by marcosmarxm** [comment]
@TLazarevic the overall goal of this one is to understand if the existing Mixpanel connector works in PyAirbyte. If it doesn't report what issues you faced.

**Comment by marcosmarxm** [comment]
Let me know if you need any further input here

**Comment by btkcodedev** [comment]
I would like to take up this issue, if there is no assignee

**Comment by avirajsingh7** [comment]
@marcosmarxm can you assign me this one, @btkcodedev already assigned to #32

**Comment by TLazarevic** [comment]
I am working on this one, what input do you need from me?

**Comment by marcosmarxm** [comment]
@TLazarevic re-assigned to you!

**Comment by Mitep** [comment]
Hello guys,
we've made some progress and we'd like to let you know.
Everything we worked on can be found under this [pull request](https://github.com/TLazarevic/airbyte-hackathon/pull/1/files).

Here you have a [doc](https://github.com/TLazarevic/airbyte-hackathon/blob/5754039ff31f3947e6f2dd76adb5924eb0efdff6/debugging/issues.md) with the outlined issues we found so far.

There are a couple of tests and some playground files which we use for debugging.

Expect more in the upcoming days.
